chat-widget/src/components/headerstateful/HeaderStateful.tsx

```diff
@@ -24,7 +24,7 @@ export const HeaderStateful = (props: IHeaderStatefulParams) => {
     const [outOfOperatingHours, setOutOfOperatingHours] = useState(state.domainStates.liveChatConfig?.LiveWSAndLiveChatEngJoin?.OutOfOperatingHours === "True");
 
     const outOfOfficeStyleProps: IHeaderStyleProps = Object.assign({}, defaultOutOfOfficeHeaderStyleProps, outOfOfficeHeaderProps?.styleProps);
-
+    const conversationState = useRef(state.appStates.conversationState);
```

Comment:
why is this needed?

Reply:
This is added to immediately update the controlProps in Header once there is a change with the app conversationState without triggering a re-render of the component